frontend/src/model/beta/reviews/ApprovalsDisplay.tsx

```diff
@@ -0,0 +1,50 @@
+import { Done } from '@mui/icons-material'
+import { Stack, Tooltip, Typography } from '@mui/material'
+import { useGetModelRoles } from 'actions/model'
+import { useMemo } from 'react'
+import Loading from 'src/common/Loading'
+import MessageAlert from 'src/MessageAlert'
+import { ReviewResponseWithRole } from 'types/interfaces'
+import { plural } from 'utils/stringUtils'
+
+interface ApprovalsDisplayProps {
+  modelId: string
+  acceptedReviewResponses: ReviewResponseWithRole[]
+}
+
+export default function ApprovalsDisplay({ modelId, acceptedReviewResponses }: ApprovalsDisplayProps) {
+  const { modelRoles, isModelRolesLoading, isModelRolesError } = useGetModelRoles(modelId)
+
+  const roleApprovals = useMemo(() => {
+    const staticRoles = ['owner', 'contributor', 'consumer']
+    const dynamicRoles = modelRoles.filter((role) => !staticRoles.includes(role.id)).map((role) => role)
```

Comment:
`.filter()` already returns a new array of the filtered items, so this `.map()` isn't doing anything.

I've removed it from the code snippet in my other comment, but thought it worth pointing out.